mani_skill/utils/building/actor_builder.py

```diff
@@ -185,10 +190,33 @@ def build(self, name):
         else:
             self.scene_idxs = torch.arange((self.scene.num_envs), dtype=int)
         num_actors = len(self.scene_idxs)
-        initial_pose = Pose.create(self.initial_pose)
-        initial_pose_b = initial_pose.raw_pose.shape[0]
+
+        if self.initial_pose is None:
+            if self.physx_body_type == "static":
+                warnings.warn(
```

Comment:
import logger and warn from there instead, it is scoped

Reply:
using logger.warn now